pkg/apis/serving/v1/route_validation.go

```diff
@@ -146,7 +146,7 @@ func (tt *TrafficTarget) validateLatestRevision(ctx context.Context) *apis.Field
 		if pinned == lr {
 			// The senses for whether to pin to a particular revision or
 			// float forward to the latest revision must match.
-			return apis.ErrInvalidValue(lr, "latestRevision")
+			return apis.ErrGeneric(fmt.Sprintf("May not set revisionName %q when latestRevision is %t", tt.RevisionName, lr), "latestRevision")
```

Comment:
Errors should always start with a lowercase character (I can't my source for it 😂, maybe you just trust me?). It's to aid readability on composed errors.

Reply:
Yes, the log should always start with a lowercase character (I also cannot find the source, but it should be.) Thank you. Updated.